packages/gluestick/src/config/webpack/webpack.config.client.js

```diff
@@ -64,35 +72,5 @@ module.exports = (
     config.plugins.push(progressHandler(logger, 'client'));
   }
 
-  // If vendor Dll bundle exists, use it otherwise fallback to CommonsChunkPlugin.
-  const vendorDllManifestPath: string = path.join(
```

Comment:
We should make a decision on whether to keep all the unused DLL code or not. There is a lot of it.